Makefile

```diff
@@ -10,6 +10,11 @@ FLAGS = save_datastore disable_host_checking no_auto_restart prod_env maintenanc
 
 sharding_instances := 3
 
+# This escapes any special characters and remove any spaces in the PATH. We do this to resolve errors in WSL due to windows paths.
+# Path like this /usr/git cli/git -> /usr/gitcli/git this is temporary for this Makefile processes.
```

Comment:
Can you remove this example, I don't think we especially need it here.

Reply:
I'll do it